icgcget/cli.py

```diff
@@ -26,9 +26,10 @@
 from clients.gdc import gdc_client
 from clients.gnos import gt_client
 from clients.icgc import icgc_client
-from utils import flatten_dict, normalize_keys
+from clients.icgc import icgc_api
+from utils import flatten_dict, normalize_keys, match_repositories
 
-REPOS = ['collab', 'aws', 'ega', 'gdc', 'cghub']
+REPOS = ['collaboratory', 'aws-virginia', 'ega', 'gdc', 'cghub']  # Updated for codes used by api
```

Comment:
Strictly speaking, neither `collab` or `aws` are mentioned in the config or options, they both share the `icgc` name.  So the most consistent way to use them would be for our tool to use the same language as the portal and api: which are these terms.